openadapt/db.py

```diff
@@ -30,15 +39,24 @@ def __repr__(self):
         return f"{self.__class__.__name__}({params})"
 
 
-def get_engine():
+def get_engine() -> Any:
```

Comment:
Isn't the return type the same as that returned by `sqlalchemy.create_engine` (https://docs.sqlalchemy.org/en/20/core/engines.html#sqlalchemy.create_engine) ?

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507

>